pkg/azurefile/controllerserver.go

```diff
@@ -136,6 +148,10 @@ func (d *Driver) CreateVolume(ctx context.Context, req *csi.CreateVolumeRequest)
 				klog.Warningf("CreateFileShare(%s) on account(%s) failed with error(%v), sleep 3s to retry", validFileShareName, account, retErr)
 				time.Sleep(3 * time.Second)
 				return false, nil
+			} else if strings.Contains(retErr.Error(), shareBeingDeleted) {
```

Comment:
need to check whether we could check return code instead of error msg later

Reply:
looks like it only returns error msg:
https://github.com/Azure/azure-sdk-for-go/blob/4f14e3d53fba12172734d383af647730d06412d2/services/storage/mgmt/2019-06-01/storage/fileshares.go#L56

Reply:
Yep. All apis in azure-sdk-for-go just return error msg.